compiler_base/span/src/lib.rs

```diff
@@ -9,13 +9,27 @@
 
 pub mod span;
 pub use rustc_span::fatal_error;
+use span::new_byte_pos;
 pub use span::{BytePos, Span, SpanData, DUMMY_SP};
 
 pub type SourceMap = rustc_span::SourceMap;
 pub type SourceFile = rustc_span::SourceFile;
 pub type FilePathMapping = rustc_span::source_map::FilePathMapping;
 pub type Loc = rustc_span::Loc;
 
+/// New a `Span`.
+///
+/// # Examples
+///
+/// ```
+/// use compiler_base_span::new_span;
+/// let span = new_span(0, 0);
+/// ```
+#[inline]
+pub fn new_span(lo: u32, hi: u32) -> Span {
+    Span::new(new_byte_pos(lo), new_byte_pos(hi))
```

Comment:
Why isn't this a method for Span? Can `new_byte_pos` be an Int method implementation?